packages/gravity-ui-nunjucks/__tests__/a11y.spec.js

```diff
@@ -0,0 +1,73 @@
+const AxeBuilder = require('axe-webdriverjs');
+const selenium = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const { getComponentsNames, getViolations } = require('./_helpers');
+const pkgEnvs = require('../build/envs');
+const pkgPaths = require('../build/paths');
+
+const excludedFiles = [
+  '_preview',
+  'about',
+  '404-error-page',
+  'navigation',
+  'article',
+  'symbols',
+  'svg-symbols',
+  'image',
+  'inline-svg-symbol',
+  'logotype',
+  'logo',
+  'wiprodigital',
+  'designit',
+  'icon',
+];
+
+describe('A11y tests', () => {
+  let driver;
+  const components = getComponentsNames(pkgPaths.srcComponentsPath(), excludedFiles, '.njk');
+
+  components.forEach((component) => {
+    describe(`${component} component`, () => {
+      beforeEach((done) => {
+        driver = new selenium.Builder()
```

Comment:
Is it possible to move the setup steps into a `beforeAll()`, so that the browser is only launched once? Presumably the setup for each test only needs to be constructing the URL and calling `driver.get(...)` to navigate to the new page.

This might speed up the test runs a bit.

Reply:
`beforeAll()` was added and changed from scope in order to launch the browser once. Also it was changed the library. Instead of using Selenium, now we are using Puppeteer. So, installing a Chrome driver is not required.